packages/vuetify/src/components/VAutocomplete/VAutocomplete.ts

```diff
@@ -44,9 +44,9 @@ export default VSelect.extend({
     },
     noFilter: Boolean,
     searchInput: {
-      type: String as PropType<string | undefined>,
+      type: String,
       default: undefined,
-    },
+    } as PropValidator<string | undefined>,
```

Comment:
Why? We've been slowly moving to PropType since it was added.

Reply:
Ah ok, I had some problem when I used PropType, so I've changed to PropValidator in few places, but it appeared that the problem was unrelated to it. 
I'll change all PropValidator to PropType then